Update om3 config doc for 25km ryf 1.0-beta release (#644)
COSIMA/MOM6-CICE6: documentation/docs/pages/Architecture.md

The schematic below illustrates the structure of the MOM6-CICE6-WW3 ACCESS-OM3 executable. ACCESS-OM3 is a single executable, consisting of the NUOPC driver (the main program) and several model components, each wrapped in a NUOPC cap; the caps are coupled through the CMEPS mediator via NUOPC connectors (see the [coupling page](Coupling.md) for more information).

![ACCESS-OM3 architecture](../assets/nuopc_overview.png){: loading="lazy" }

The coupled fields and remapping methods used are recorded in the mediator log output file and can be found with `grep '^ mapping' archive/output000/log/med.log`; see [here](https://escomp.github.io/CMEPS/versions/master/html/esmflds.html) for how to decode this. See [the Configurations Overview page](configurations/Overview.md#coupling) for details on how the coupling is determined.

## Overview of codebase

> The ACCESS-OM3 code repository consists mostly of submodules containing the code for each model component.

Reviewer: ```suggestion
The ACCESS-OM3 software is built from libraries containing the code for each model component.
```
